html/janus.js

```diff
@@ -37,13 +37,112 @@ Janus.isExtensionEnabled = function() {
 			// Older versions of Chrome don't support this extension-based approach, so lie
 			return true;
 		}
-		return ($('#janus-extension-installed').length > 0);
+		return Janus.checkJanusExtendsion();
```

Comment:
Typo? (s/Extendsion/Extension). There seems to be the same typo in all the code.